x-pack/plugins/ui_actions_enhanced/public/drilldowns/drilldown_manager/containers/action_factory_picker/action_factory_picker.tsx

```diff
@@ -14,14 +14,17 @@ export const ActionFactoryPicker: React.FC = ({}) => {
   const drilldowns = useDrilldownManager();
   const factory = drilldowns.useActionFactory();
   const context = React.useMemo(() => drilldowns.getActionFactoryContext(), [drilldowns]);
+  const applicableFactories = drilldowns.deps.actionFactories.filter((actionFactory) =>
+    actionFactory.isConfigurable(context)
```

Comment:
Shouldn't this be 

```suggestion
    actionFactory.isCompatible(context)
```

and we would not need the new `isConfigurable` property? Or am I missing something. What is the difference between `isCompatible` and `isConfigurable` for the action factory?

Reply:
>  @vadimkibana what do you think about this extension? I'm sure it will be useful for other things as well.

I'm trying to understand how `isConfigurable` is different from the existing `isCompatible`.

Reply:
I wasn't sure about the semantics of `isCompatible` in this place, but it seems like it's coming from here https://github.com/elastic/kibana/blob/3730dd0779ed8efd74aee88f57422781ec1ac122/src/plugins/ui_actions/public/util/presentable.ts#L56

- in this case it's a pretty good fit. Will adjust